materialsystem/shaderapidx9/shaderdevicedx8.cpp

```diff
@@ -3451,6 +3456,7 @@ void CShaderDeviceDx8::Present()
 			{
 				hr = Dx9Device()->Present(0, 0, 0, 0);
 			}
+			#endif

```

Comment:
This skips presenting on OpenGL. put the #endif on a new line after 3455 (the else). That way, the if and else will be removed and you're left with the else block.

Reply:
Fixed.